presentation-compiler/src/main/dotty/tools/pc/completions/NamedArgCompletions.scala

```diff
@@ -63,9 +67,29 @@ object NamedArgCompletions:
             Some(ident),
             app,
             indexedContext,
-            clientSupportsSnippets
+            clientSupportsSnippets,
           )
         contribution.getOrElse(Nil)
+      case (app: Apply) :: _ =>
+        /**
+         * def foo(aaa: Int, bbb: Int, ccc: Int) = ???
+         * val x = foo(
+         *  bbb = 123,
+         *  ccc = 123,
+         *  @@
+         * )
+         * In this case, typed path doesn't contain already provided arguments
+         */
+        NavigateAST.untypedPath(pos.span) match
```

Comment:
Untyped path is already computed for the completions. We shouldn't recompute it. Check for 
[val adjustedPath = Completion.resolveTypedOrUntypedPath(path, pos)](https://github.com/jkciesluk/dotty/blob/2849c1fa9d95e97569d7adf394568dfe1c800b75/presentation-compiler/src/main/dotty/tools/pc/completions/Completions.scala#L60)